workflow-templates/git-slice-v2.yml

```diff
@@ -44,17 +44,37 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.event_name == 'workflow_dispatch' }}
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out repository
+        uses: actions/checkout@v2
         with:
           fetch-depth: 1
+
+      - name: Setup Git
+        run: |
+          git config --global user.email ${{ secrets.EMAIL }}
+          git config --global user.name  ${{ secrets.USERNAME }}
+          git config --global pull.rebase true
+          
```

Comment:
will require team leads/csms to add the email of the gitstart bot account respective to the client